File: Scripts/laptop/json/javascript/json.js
var r = 1;
function executeTask(i) {

var data = JSON.parse('{ "foo": 1, "bar": [10, "apples"] }');

var sample = { "blue": [1,2], "ocean": "water" };
var json_string = JSON.stringify(sample);
return i + 1;
}

for ( var i = 0; i < 10000000; ++i) {
	r = executeTask(i + r);
}

File: Scripts/laptop/json/go/json.go
package main

import "encoding/json"

func executeTask(i int) int {

    		var data interface{}
    		jsonString := json.Unmarshal([]byte(`{"foo":1, "bar":[10, "apples"]}`), &data)
		_ = jsonString
return i + 1
}

func main() {
	r := int(1)
	for i := 0; i < 10000000; i++ {
		r = executeTask(i + r)
	}
	_=r
}
